bin/cellranger/cellranger_count.sh

```diff
@@ -12,12 +12,13 @@
 #   <cpu>         - Number of CPU cores.
 #   <mem>         - Memory in MB.
 #   --no-bam      - Optional flag to disable BAM file generation.
+#   --chemistry <value> - Chemistry of assay kit used.
```

Comment:
You should also add debug information below and print out the chemistry. 

Below this line:
`[ -n "$BAM_FLAG" ] && echo "BAM output is disabled"`

Add this line:
`[ -n "$CHEMISTRY" ] && echo "Using chemistry option: $CHEMISTRY"`